src/raftstore/store/worker/compact.rs

```diff
@@ -70,11 +71,25 @@ quick_error! {
 
 pub struct Runner {
     engine: Arc<DB>,
+
+    // region_id -> last_compacted
+    raft_compacted_ctx: HashMap<u64, u64>,
```

Comment:
I don't like the design. We should keep Runner stateless. What Runner needs to do is get a task, and finish it. It shouldn't keep the context.